src/scss/_forms.scss

```diff
@@ -31,3 +35,15 @@
         }
     }
 }
+
+.fixed-alert {
```

Comment:
@DarrenGalway I might be missing something in a build step or something, the alerts aren't displaying properly for me:
![filamynt form](https://user-images.githubusercontent.com/2477338/32357108-04a0c0c2-c010-11e7-9982-de0881568b54.gif)